client/tx/legacy_test.go

```diff
@@ -15,10 +15,10 @@ import (
 	"github.com/cosmos/cosmos-sdk/testutil/testdata"
 	"github.com/cosmos/cosmos-sdk/types"
 	signing2 "github.com/cosmos/cosmos-sdk/types/tx/signing"
+	"github.com/cosmos/cosmos-sdk/x/auth/legacy/legacytx"
```

Comment:
How about taking this as an opportunity to move this package into some `x/auth` subdirectory so that we could break the `client/*` -> `x/*` dependency links?

Reply:
Great idea.
I would prefer to do it as a separate task. Long hanging PRs are not good.
I created an issue for that: https://github.com/cosmos/cosmos-sdk/issues/7359